android/anc/src/main/java/org/smartregister/fhircore/anc/ui/login/LoginActivity.kt

```diff
@@ -20,14 +20,13 @@ import android.content.Intent
 import android.os.Bundle
 import org.smartregister.fhircore.anc.BuildConfig
 import org.smartregister.fhircore.anc.R
-import org.smartregister.fhircore.anc.ui.family.register.FamilyRegisterActivity
 import org.smartregister.fhircore.engine.configuration.view.loginViewConfigurationOf
 import org.smartregister.fhircore.engine.ui.login.BaseLoginActivity
 
 class LoginActivity : BaseLoginActivity() {
 
   override fun navigateToHome() {
-    val intent = Intent(this, FamilyRegisterActivity::class.java)
+    val intent = Intent(this, SplashActivity::class.java)
```

Comment:
`SplashActivity` not necessary

Reply:
Would remove it after loader is done

Reply:
Kindly remove this implementation. The [LoaderView](https://github.com/opensrp/fhircore/blob/main/android/engine/src/main/java/org/smartregister/fhircore/engine/ui/components/LoaderView.kt) was implemented based on the mockup. The mockup does not have a splash screen between the login and register views. The idea is okay but it's already covered. The loader view can be activated once the issue with sync is resolved.